pkgs/tools/typesetting/sile/default.nix

```diff
@@ -1,23 +1,23 @@
 { stdenv, darwin, fetchurl, makeWrapper, pkgconfig
 , harfbuzz, icu
 , fontconfig, lua, libiconv
-, makeFontsConf, gentium, gentium-book-basic, dejavu_fonts
+, makeFontsConf, gentium
 }:
 
 with stdenv.lib;
 
 let
-  luaEnv = lua.withPackages(ps: with ps;[ lpeg luaexpat lua-zlib luafilesystem luasocket luasec]);
+  luaEnv = lua.withPackages(ps: with ps;[cassowary linenoise lpeg lua-zlib lua_cliargs luaepnf luaexpat luafilesystem luarepl luasec luasocket stdlib vstruct]);
```

Comment:
I believe some of these packages cassowary/linenoise should be added to nixpkgs since they don't appear in pkgs/development/lua-modules/generated-packages.nix